### DISCUSSION TITLE
showAddress doesn't dislpay address on card

### DISCUSSION THREAD

**Discussion by AKuzmichkin** [CoolBitX-Technology/coolwallet-signing-tools#94] [Q&A] [answered]
- Description

showAddress doesn't dislpay address on the card
Checked (also) example with Polkadot in Script Tester readme - no address on the card.
Version: 323

- Expected Behavior

showAddress displays address on the card

**Comment by hank121314** [comment, marked as answer]
Please use `@coolwallet/sdk/packages/coin-tester` and press `Toggle showDetail`.
![image](https://user-images.githubusercontent.com/17615229/164132839-9370d1ef-e867-4fa5-a20b-82e9e4ca55ba.png)
